### PR TITLE
feat: handle correctly when replica count is set to 0 

### PR DESCRIPTION
**Pull request by Tchoupinax**
Hello,

This evolution brings the ability to set the replica count to zero. There is an unexpected behaviour in helm that does not allow it as we would expect it.
My current usage of putting replica count to zero is for https://github.com/acouvreur/sablier

Thank for your work!

### CODE REVIEW

**Comment by brunoocasali** [comment]
@meilisearch sync-manifest

**Comment by brunoocasali** [comment]
Hey @Tchoupinax can you fix the conflicts?

**Comment by brunoocasali** [comment]
@meilisearch sync-manifest

**Review by brunoocasali — APPROVED** [review]

File: manifests/meilisearch.yaml [hunk 1]
```diff
@@ -60,7 +60,9 @@ metadata:
     app.kubernetes.io/component: search-engine
     app.kubernetes.io/part-of: meilisearch
 spec:
-  replicas: 1
+  # Value 0 is bad interpreted it goes to 1 by default
+  # https://github.com/helm/helm/issues/3164
+  replicas: 0
```
Comment (brunoocasali) [inline manifests/meilisearch.yaml:65]: I think we should keep this as 1 by default.

**Review by brunoocasali — CHANGES_REQUESTED** [review]

File: manifests/meilisearch.yaml (hunk 1 above)
Comment (Tchoupinax) [reply manifests/meilisearch.yaml:65]: Hello, yes it's a mistake from my side when I tested, I will revert

**Comment by Tchoupinax** [comment]
> Hey @Tchoupinax can you fix the conflicts?

Sorry just got your message. It's done now!

**Comment by brunoocasali** [comment]
bors try

**Comment by brunoocasali** [comment]
@meilisearch sync-manifest

**Comment by brunoocasali** [comment]
bors merge

**Comment by brunoocasali** [comment]
@meilisearch sync-manifest

File: manifests/meilisearch.yaml (hunk 1 above)
Comment (brunoocasali) [inline manifests/meilisearch.yaml:65]:
```suggestion
  replicas: 1
```

**Comment by brunoocasali** [comment]
@meilisearch sync-manifest

**Comment by brunoocasali** [comment]
bors merge

File: charts/meilisearch/Chart.yaml
Comment (brunoocasali) [inline charts/meilisearch/Chart.yaml:1]: @Tchoupinax I can't look into why the CI is not working right now. Can you take a look at it, please?

**Review by brunoocasali — CHANGES_REQUESTED** [review]

**Comment by brunoocasali** [comment]
@meilisearch sync-manifest

**Review by brunoocasali — APPROVED** [review]

**Pull request closed without merging**